driver/src/test/java/org/neo4j/driver/TransactionConfigTest.java

```diff
@@ -98,6 +86,27 @@ void shouldHaveTimeout()
         assertEquals( Duration.ofSeconds( 3 ), config.timeout() );
     }
 
+    @Test
+    void shouldAllowDefaultTimeout()
+    {
+        TransactionConfig config = TransactionConfig.builder()
+                .withTimeout( TransactionConfig.Builder.SERVER_DEFAULT_TIMEOUT )
+                .build();
+
+        assertNull( config.timeout() );
+        assertEquals( TransactionConfig.Builder.SERVER_DEFAULT_TIMEOUT,config.timeout() );
```

Comment:
Code formatter should have added space between arguments. (it might be worth checking IDE configuration)

Reply:
Yip, I should :sweat_smile: